bindings/gumjs/gumv8file.cpp

```diff
@@ -297,8 +303,9 @@ GUMJS_DEFINE_CLASS_METHOD (gumjs_file_read_bytes, GumFile)
   auto result = ArrayBuffer::New (isolate, n);
   auto store = result.As<ArrayBuffer> ()->GetBackingStore ();
 
-  size_t num_bytes_read = fread (store->Data (), 1, n, self->handle);
+  GumV8InterceptorIgnoreScope interceptor_ignore_scope;
 
+  size_t num_bytes_read = fread (store->Data (), 1, n, self->handle);
```

Comment:
Not sure what changed on this line?

Reply:
lol idk, probably just the line number of it